modules/plugins/jboss-as-7/src/main/java/org/rhq/modules/plugins/jbossas7/helper/ServerPluginConfiguration.java

```diff
@@ -198,15 +201,35 @@ public void setLogDir(File logDir) {
         this.pluginConfig.setSimpleValue(Property.LOG_DIR, (logDir != null) ? logDir.toString() : null);
     }
 
+    @Deprecated
```

Comment:
Can you add some Javadoc to indicate the version from which this method is deprecated? Same applies to other deprecated methods.